test/integration/clinicians/clinicians-all.js

```diff
@@ -190,6 +190,11 @@ context('clinicians list', function() {
       .visit('/clinicians')
       .wait('@routeClinicians');
 
+    cy
+      .get('.list-page__header')
+      .find('[data-search-region] .js-input:disabled')
+      .should('have.attr', 'placeholder', 'Find in List...');
+
```

Comment:
This just verifies that the `Find in list...` input is disabled when the clinicians data is done being loaded, but empty.